examples/GPEmulator/plot_GP.jl

```diff
@@ -0,0 +1,94 @@
+# Import modules
+using Random
+using GaussianProcesses
+using Distributions
+using Statistics
+using Plots; pyplot(size = (1000, 400))
+
+using CalibrateEmulateSample.GPEmulator
+
+###############################################################################
+#                                                                             #
+#   This examples shows how to fit a Gaussian Process regression model to     #
+#   2D input data, and how to plot the mean and variance                      #
+#                                                                             #
+###############################################################################
+
+function meshgrid(vx::AbstractVector{T}, vy::AbstractVector{T}) where T
+    m, n = length(vy), length(vx)
+    gx = reshape(repeat(vx, inner = m, outer = 1), m, n)
+    gy = reshape(repeat(vy, inner = 1, outer = n), m, n)
+
+    return gx, gy
+end
+
+gppackage = GPJL()
+pred_type = YType()
+
+# Seed for pseudo-random number generator
+rng_seed = 41
+Random.seed!(rng_seed)
+
+
+# Generate training data (x-y pairs, where x ∈ ℝ ᵖ, y ∈ ℝ ᵈ)
+# x = [x1, x2]: inputs/predictors/features/parameters
+# y = [y1, y2]: outputs/predictands/targets/observables
+# The observables y are related to the parameters x by: 
+# y = G(x1, x2) + η, 
+# where G(x1, x2) := [sin(x1) + cos(x2), sin(x1) - cos(x2)], and η ~ N(0, Σ)
+n = 100 # number of training points
+p = 2   # input dim 
+d = 2   # output dim
+X = 2.0 * π * rand(n, p)                        
+μ = zeros(d) 
+Σ = 0.1 * [[0.5, 0.2] [0.2, 0.8]] # d x d
+
+g1 = sin.(X[:, 1]) .+ cos.(X[:, 2])
+g2 = sin.(X[:, 1]) .- cos.(X[:, 2])
+Y = [g1 g2] .+ rand(MvNormal(μ, Σ), n)' 
+truth_cov = cov(Y, dims=1)
+
+# Fit 2D Gaussian Process regression model
+# (To be precise, we fit two models, one that predicts y1 from x1 and x2, 
+# and one that predicts y2 from x1 and x2)
+gpobj = GPObj(X, Y, truth_cov, gppackage, GPkernel=nothing, 
+              normalized=true, noise_learn=true, prediction_type=pred_type)
```

Comment:
Since this is a tutorial for fitting a Gaussian process, it would be nice to either discuss what the default GPKernel is or to define and use a simple one here.

Reply:
Yes, agreed. I'll add either some explanation or demonstrate how to construct your own kernel once the example works.